README.md

```diff
@@ -325,8 +326,8 @@ state as the new version of the website available under [https://wo-ist-markt.de
 [paderborn-markets]: http://www.paderborn.de/microsite/wochenmarkt/marktinfos/109010100000079411.php?p=5,1
 [potsdam-wikipedia]: https://en.wikipedia.org/wiki/Potsdam
 [potsdam-markets]: https://www.potsdam.de/de/kategorie/einkaufen-schenken
-[rheinstetten-wikipedia]: https://de.wikipedia.org/wiki/Rheinstetten
-[rheinstetten-markets]: https://www.rheinstetten.de/de/leben-in-rheinstetten/freizeit/wochenmaerkte
```

Comment:
Rheinstetten is broken in your commit. See here in the [README rendering](https://github.com/Marwe/wo-ist-markt.github.io/blob/5f08e617cee01781c92b9b1ac572bba527c1dd4c/README.md#supported-cities)

![image](https://github.com/user-attachments/assets/cc4c038f-04de-4330-80cc-83ea261ef83e)

Reply:
@Marwe Did you see my review here?

Reply:
Saw it today and corrected the missing entries. Checked some validation errors, too.

These issues remain in the validation, ACCEPTABLE_WARNINGS_COUNT=2

# Marl 

## validate: IPv6 dysfunct

```
Warning: marl: Error while accessing data source url: connect EHOSTUNREACH 2a02:100e:bef8:2:0:57:135:1:443

Resolving www.marl.de (www.marl.de)... 2a02:100e:bef8:2:0:57:135:1, 185.41.57.135
Connecting to www.marl.de (www.marl.de)|2a02:100e:bef8:2:0:57:135:1|:443... failed: No route to host.
```

IPv6 not properly configured

https://ready.chair6.net/?url=https%3A%2F%2Fwww.marl.de

Maybe due to https://konbriefing.com/de-topics/cyberangriff-2023-sit-suedwestfalen-it.html - via https://kommunaler-notbetrieb.de/2023/10/30/zweckverband-suedwestfalen-it/

# Chemnitz

Ernst-Enge-Straße only 2024, 2025 to be removed

## validate: user agent rejected

wget -U "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/115.0" -O /dev/null https://www.chemnitz.de/de/rathaus/ausschreibungen/marktausschreibungen/wochenmaerkte
--2024-12-10 19:18:27--  https://www.chemnitz.de/de/rathaus/ausschreibungen/marktausschreibungen/wochenmaerkte
Resolving www.chemnitz.de (www.chemnitz.de)... 3.76.143.127, 18.195.160.253
Connecting to www.chemnitz.de (www.chemnitz.de)|3.76.143.127|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 612964 (599K) [text/html]
Saving to: ‘/dev/null’

Reply:
This looks reasonable to me. Thank you for taking the time to investigate :thum